spec/features/admin/accounts_spec.rb

```diff
@@ -35,7 +35,7 @@
     click_on "Accounts"
 
     page.driver.browser.manage.window.resize_to(1024, 768)
-    page.driver.save_screenshot Rails.root.join("app/assets/images/screenshots/create_account_01.png"), full: true
+    page.driver.save_screenshot Rails.root.join("app/assets/images/screenshots/create_account_01.png")
```

Comment:
I'm wondering why are you saving these screenshots. :thinking:

Reply:
> I'm wondering why are you saving these screenshots. 🤔

There are some docs in the `/docs` [dir](https://github.com/dmitryrck/organize2/tree/master/docs) of this repo, but I never created the website.